README.md

```diff
@@ -22,15 +22,21 @@ please refer to the [AOSC OS Maintenance Guidelines](https://wiki.aosc.io/develo
 Groups
 ------
 
-When using [ACBS](https://github.com/AOSC-Dev/acbs/) (Autobuild CI Build System),
+When using [ACBS](https://github.com/AOSC-Dev/acbs) (Autobuild CI Build System),
 it is possible to build a series of packages with a single command. This is
 achieved by the "groups" defined in the `groups/` directory, with files defining
 a list of packages to build.
 
-Automation
-----------
+Notes for developers on Windows
+-------------------------------
 
-Under GitHub issues and pull requests, you may initiate package build tests via GitHub Actions, see [reference of workflows](.github/workflows/README.md) for details.
+Currently, this repository cannot be checked out on Windows, since there're
+backslashes in file names, specifically required by [systemd.mount](https://www.freedesktop.org/software/systemd/man/latest/systemd.mount.html#Description)
+for [string escaping](https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#String%20Escaping%20for%20Inclusion%20in%20Unit%20Names),
+which is [not allowed by Windows](https://learn.microsoft.com/windows/win32/fileio/naming-a-file#naming-conventions).
```

Comment:
not a legal file name in Windows filesystems

Reply:
The Style Guide asks to replace "legal" with "valid". Also, it is a valid file name in NTFS or ReFS (I think), but not a valid file name in Windows (neither NT nor Win32 I think).

Reply:
Sure, so:

not a valid Windows file name

Reply:
Resolved.